code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -9,7 +9,7 @@
 	var/obj/machinery/camera/current = null
 
 	var/ram = 100	// Used as currency to purchase different abilities
-	var/list/software = list()
+	var/list/software = list("crew manifest","digital messenger")

```

Comment:
That's why you use defines everywhere.
There wouldn't be much point to a string define if it was only used once.